packages/infra/dashboards/cw-dashboard-prod.json

```diff
@@ -389,8 +389,8 @@
                 "annotations": {
                     "horizontal": [
                         {
-                            "label": "VolumeWriteIOPs >= 5000 for 1 datapoints within 5 minutes",
-                            "value": 5000,
+                            "label": "VolumeWriteIOPs >= 10_000 for 1 datapoints within 5 minutes",
+                            "value": 10000,
```

Comment:
That should be 100K, right?

Reply:
omg.. yes, now I finally see how one is for fhir-server and the other is for api-cluster all in the same file! pfff thank you!